deployment/.gitignore

```diff
@@ -0,0 +1,2 @@
+postgres/
```

Comment:
So I've just merged #40 and it brought some conflicts. Besides, due to https://github.com/serokell/edna/commit/f65cd28c1d042a0eb34e22802b876761abb4578c we don't create `postgres/` directory here anymore, so you can drop this line.